src/index.js

```diff
@@ -11,6 +11,11 @@ import {store, start as startStore} from './store/store.js';
 import log from 'electron-log';
 import 'roboto-fontface/css/roboto/sass/roboto-fontface.scss'
 import './index.scss'
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+const muiTheme = getMuiTheme({
+    fontFamily: 'Poppins Regular',
+});
```

Comment:
According to the Invision thing, the font is "GT Walsheim Pro". Is "Poppins" close enough? If so, I will update my changes to use your font.